default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> It doesn't download those by the way, it just copies itself to those folders so that they are Protobuild modules

Yeah... but you have to agree how ridiculous that is getting.

> This is detected based on the version of MSBuild you have installed

Ah... that is right.  I'll try again on my laptop to be sure.

> It's no different to the NuGet restore (except that this is explicit references instead of implied).

But it is a mess and unnecessary...  which is why we don't want it in there.

> This is a new Protobuild feature that allows you to define post-build hooks for projects.

Ok...  i'll keep an eye out for it.

> I should note that the project Protobuild is producing actually works, so the extra references aren't a concern because they are what is making the project actually build

Works isn't enough here... we want it to work right.

Once I work out what the issue was we can figure out the next step, but likely it will be begin by removing that `<Package>` reference for `UniversalWindowsPlatform`.